src/luascript.cpp

```diff
@@ -504,14 +504,14 @@ int LuaScriptInterface::luaErrorHandler(lua_State* L)
 	return 1;
 }
 
-bool LuaScriptInterface::callFunction(int params)
+bool LuaScriptInterface::callFunction(int params, bool defaultReturn /* = false */)
```

Comment:
Sorry if I'm being too pedantic here, but wouldn't it be nice to have standard parameter name? Like using `defaultValue` everywhere.